core/src/main/java/com/novoda/noplayer/internal/exoplayer/CompositeTrackSelectorCreator.java

```diff
@@ -25,6 +25,10 @@ CompositeTrackSelector create(Options options, DefaultBandwidthMeter bandwidthMe
                 Clock.DEFAULT
         );
         DefaultTrackSelector trackSelector = new DefaultTrackSelector(adaptiveTrackSelectionFactory);
+        DefaultTrackSelector.Parameters trackSelectorParameters = trackSelector.buildUponParameters()
+                .setMaxVideoBitrate(options.maxVideoBitrate())
```

Comment:
We initialise the `DefaultTrackSelector` with the known `maxVideoBitrate` from the `options`.